### PR TITLE
Randomization methods

### PR DESCRIPTION
**Pull request by danamansana**
This adds random_shuffle, uniform, and none as possible randomization methods, configurable in the environment variables.

### CODE REVIEW

File: sam.local.yml [hunk 1]
```diff
@@ -20,4 +20,5 @@ Resources:
           S3_FEED_KEY: 'feed.xml'
           LOG_LEVEL: debug
           TZ: 'America/New_York'
+          RANDOMIZATION_METHOD: 'uniform'
```
Comment (ktp242) [inline sam.local.yml:23]: Probably document some methods for randomization here? I still am not sure how many variations we have here.

**Review by nonword — APPROVED** [review]
Looks great. As discussed just needs some added comments and perhaps a slight refactor to the randomization util

File: lib/item_stream_handler.rb [hunk 2]
```diff
@@ -61,9 +62,10 @@ def handle (event)
     update_tally_if_necessary
     checkout_count = 0
 
-    event["Records"]
+    records = event["Records"]
       .select { |record| record["eventSource"] == "aws:kinesis" }
-      .each do |record|
+    records = PreProcessingRandomizationUtil.send(ENV['RANDOMIZATION_METHOD'], records)
```
Comment (ktp242) [inline lib/item_stream_handler.rb:67]: Does this mean you send the second parameter (here it is `records`) to the first parameter (here it is `ENV['RANDOMIZATION_METHOD']`) ? You send the method to the class `PreProcessingRandomizationUtil` but I don't see any method in `PreProcessingRandomizationUtil` will take the value of `ENV['RANDOMIZATION_METHOD']`

**Comment by ktp242** [comment]
Hey I still have some questions for the PR! Also I think this is kind of a big change so worths to be added in CHANGE.log

**Pull request merged**